src/js/utils/log.js

```diff
@@ -16,27 +16,30 @@ import {IE_VERSION} from './browser';
  *         but this is exposed as a parameter to facilitate testing.
  */
 export const logByType = (type, args, stringify = !!IE_VERSION && IE_VERSION < 11) => {
-  const console = window.console;
 
-  // If there's no console then don't try to output messages, but they will
-  // still be stored in `log.history`.
-  //
-  // Was setting these once outside of this function, but containing them
-  // in the function makes it easier to test cases where console doesn't exist
-  // when the module is executed.
-  const fn = console && console[type] || function(){};
+  // Make sure this is a plain old array.
+  args = Array.prototype.slice.call(args);
 
+  // Add the type to the front of the message when it's not "log"
   if (type !== 'log') {
-
-    // add the type to the front of the message when it's not "log"
     args.unshift(type.toUpperCase() + ':');
   }
 
-  // add to history
+  args.unshift('VIDEOJS:');
+
+  // Add arguments to history
   log.history.push(args);
 
-  // add console prefix after adding to history
```

Comment:
It's not. I moved that line about the `history.push` call. And, since `args` is added to `history` by reference, it never mattered if this was "after" the `history.push` call.